app/views/projects/_form.html.erb

```diff
@@ -0,0 +1,65 @@
+<%= form_for @project  do |form| %>
+  <div class="container-fluid">
+    <% if @project.errors.any? %>
+      <% @project.errors.full_messages.each do |msg| %>
```

Comment:
place these error messages into flash[:error] in projects.rb(in respective actions).